src/main/java/util/TransactionValidator.java

```diff
@@ -0,0 +1,21 @@
+package util;
+
+import model.FruitTransaction;
+
+public class TransactionValidator {
+
+    private TransactionValidator() {
+        throw new UnsupportedOperationException("Utility class");
+    }
```

Comment:
Don`t do this anymore)

Reply:
ok, but why? 
Is it better to make this class static so that no object of this class can be created?

It's just logical(maybe😁) that it's like an utility class, so you don't need to make an object of this class

Reply:
private constructors :)